aws/table_aws_cloudwatch_log_group.go

```diff
@@ -200,12 +231,16 @@ func getLogGroupTagging(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
 		return nil, err
 	}
 
-	params := &cloudwatchlogs.ListTagsLogGroupInput{
-		LogGroupName: logGroup.LogGroupName,
+	// Removing extra characters from the end of Arn
+	logGroupArn := string(*logGroup.Arn)
```

Comment:
Can't we pass the log group arn as follows instead of `string(*logGroup.Arn)` ?
```suggestion
	logGroupArn := logGroup.Arn
```

Reply:
Adding a comment here to bump this question as part of my review